### PR TITLE
E2E: Module Setup Flow | Tag Manager

### PR DESCRIPTION
**Pull request by aaemnnosttv**
## Summary

Addresses issue #267 

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: tests/e2e/plugins/module-setup-tagmanager.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		array_filter(
+			$items,
+			function ( $item ) use ( $account_id ) {
+				return $item['accountId'] === $account_id;
+			}
+		)
+	);
+}
+
+//function filter_by_public_id( $items, $public_id ) {
```
Comment (swissspidy) [inline tests/e2e/plugins/module-setup-tagmanager.php]: What's with this commented out code here? Might wanna remove it if not needed.

**Comment by swissspidy** [comment]
Shouldn't this also cover the `GTM code is inserted in the home page` / `No code is inserted` parts of the ACs? Perhaps using something like this:

```js
const newPage = await browser.newPage();
const response = newPage page.goto( process.env.WP_BASE_URL );
const source = await response.text();
// Scan `source` for GTM code
```

File: tests/e2e/plugins/module-setup-tagmanager.php (hunk 1 above)
Comment (aaemnnosttv) [reply tests/e2e/plugins/module-setup-tagmanager.php]: Hmmm, I would think PHPCS should have caught that. I guess it's not running on E2E code?

Comment (swissspidy) [reply tests/e2e/plugins/module-setup-tagmanager.php]: Actually, it looks like it is not running for any test code... which is a bit  odd.

**Comment by aaemnnosttv** [comment]
@swissspidy I've updated the tests to check for the tags, although I believe there is a correction to be made to the AC for the second scenario. I've flagged this to @felixarntz [on the ticket](https://github.com/google/site-kit-wp/issues/267#issuecomment-532141373).

**Pull request merged**